backend/graphql/resolvers/testResolvers.ts

```diff
@@ -0,0 +1,84 @@
+import TestService from "../../services/implementations/testService";
+import UserService from "../../services/implementations/userService";
+import {
+  ITestService,
+  TestRequestDTO,
+  TestResponseDTO,
+  QuestionRequest,
+} from "../../services/interfaces/testService";
+import IUserService from "../../services/interfaces/userService";
+
+import {
+  MultipleChoiceMetadata,
+  NumericQuestionMetadata,
+  Question,
+} from "../../models/test.model";
+
+const userService: IUserService = new UserService();
+const testService: ITestService = new TestService(userService);
+
+const resolveQuestions = (questions: QuestionRequest[]): Question[] => {
+  const resolvedQuestions: Question[] = [];
+
+  questions.forEach((question: QuestionRequest) => {
+    const multipleChoiceMetadata: MultipleChoiceMetadata =
+      question.questionMetadataMultipleChoice;
+    const numericQuestionMetadata: NumericQuestionMetadata =
+      question.questionMetadataNumericQuestion;
+    const questionMetadata =
+      question.questionType.toString() === "MULTIPLE_CHOICE"
```

Comment:
not sure what's happening here - i think the graphql mutation passes the string and typescript implicitly converts it to an enum (since its typed as an enum)

works as is right now, but i can follow up when we use these resolvers if any bugs come up

Reply:
other option might be to change typescript types from `enum` to `string` but that's a less desirable solution